src/lib/Constants.ts

```diff
@@ -15,22 +16,19 @@ export const INTEGERS = {
 export const COLLATERAL_ASSET_ID = {
   [Networks.MAINNET]: getAssetId(usdcAbi.networks[1].address),
   [Networks.ROPSTEN]: getAssetId(usdcAbi.networks[3].address),
+  [Networks.GOERLI]: getAssetId(usdcAbi.networks[5].address),
 };
 
-export const ADDRESSES = {
-  ZERO: '0x0000000000000000000000000000000000000000',
-  DYDX_USDC_ADDRESS_ROPSTEN: '0x8707a5bf4c2842d46b31a405ba41b858c0f876c4',
-  USDC_ADDRESS_MAINNET: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
-  USDC_EXCHANGE_ADDRESS_ROPSTEN: '0x144b40e133495811e1d5fd64ab73ed40186ecb17',
-  USDC_EXCHANGE_ADDRESS_MAINNET: '0x032b17633c956c10845643f0bf9ea7c16a3cfb62',
-};
+export const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
 
 export const USDC_ADDRESSES: {[networkId: number]: string } = {
-  [Networks.ROPSTEN]: ADDRESSES.DYDX_USDC_ADDRESS_ROPSTEN,
-  [Networks.MAINNET]: ADDRESSES.USDC_ADDRESS_MAINNET,
+  [Networks.MAINNET]: usdcAbi.networks[1].address,
```

Comment:
could index into these using `Networks.MAINNET` instead of 1 3 5

Reply:
didnt seem to work for some reason